include/ensmallen_bits/cmaes/cmaes_impl.hpp

```diff
@@ -67,16 +69,16 @@ double CMAES<SelectionPolicyType>::Optimize(
 
   // Step size control parameters.
   arma::vec sigma(3);
-  sigma(0) = 0.3 * (upperBound - lowerBound);
+  sigma(0) = initialSigma;
   const double cs = (muEffective + 2) / (iterate.n_elem + muEffective + 5);
-  const double ds = 1 + cs + 2 * std::max(std::sqrt((muEffective - 1) /
+  ds = 1 + cs + 2 * std::max(std::sqrt((muEffective - 1) /
       (iterate.n_elem + 1)) - 1, 0.0);
```

Comment:
I guess, then, that if the user has set `ds` or `cc`, they get overwritten here when `Optimize()` is called?